src/yaml.cr

```diff
@@ -75,6 +75,14 @@ require "base64"
 # File.open("foo.yml", "w") { |f| {hello: "world"}.to_yaml(f) } # writes it to the file
 # ```
 module YAML
+  private alias HandlerResponse = Bool | Float64 | Int64 | String | Time | Nil
+
+  class_getter tag_handlers : Hash(String, Proc(String, HandlerResponse)) { Hash(String, Proc(String, HandlerResponse)).new }
```

Comment:
Lazy instantiation here is pointless anyway due to a `YAML.tag_handlers[tag]?` call.

```suggestion
  class_getter tag_handlers = Hash(String, Proc(String, HandlerResponse)).new
```